mysite/w6/beanstalk.py

```diff
@@ -46,21 +50,33 @@ def section_beanstalk():
 
             gold_foods[food.codename] += food.amount
 
-    foods_to_10k = [
-        k for k, v in beanstalk_status.items() if v < TEN_K and gold_foods[k] < 10**4
-    ]
-    foods_to_deposit_10k = [
-        k for k, v in beanstalk_status.items() if v < TEN_K and k not in foods_to_10k
+    foods_to_beanstack = [k for k, v in beanstalk_status.items() if v < TEN_K and gold_foods[k] < BEANSTACK_GOAL]
+    foods_to_deposit_for_beanstack = [
+        k for k, v in beanstalk_status.items() if v < TEN_K and k not in foods_to_beanstack
     ]
 
-    foods_to_100k = [
-        k for k, v in beanstalk_status.items() if v < HUNNIT_K and gold_foods[k] < 10**5
+    super_beanstack_progress: dict[str, int] = dict()
+    for gold_food_code in gold_foods:
```

Comment:
`for gold_food_code, total_owned in gold_foods.items()`

Reply:
Good catch 👍🏽